### PR TITLE
Use config id_field for SkipExistingProducts

### PR DESCRIPTION
**Pull request by vseager**
Use config id_field for SkipExistingProducts instead of hardcoded "sku". Replicated logic from SkipNonExistingProducts.

### CODE REVIEW

File: src/Filter/SkipExistingProducts.php [hunk 1]
```diff
@@ -2,16 +2,23 @@
...
 {
     /**
      * @var array
      */
     private $existingSkus = [];
 
+    /**
+     * @var string
+     */
+    private $skuField = 'sku';
```
Comment (bartoszkubicki) [inline src/Filter/SkipExistingProducts.php:20]: do we need default value if we always override it in prepare()?

Comment (vseager) [reply src/Filter/SkipExistingProducts.php:20]: I had just based this PR on the file `src/Filter/SkipNonExistingProducts.php` for consistency, but I agree, probably not.

Comment (bartoszkubicki) [reply src/Filter/SkipExistingProducts.php:20]: That module is not the best quality on all areas - development start long time ago. That's why I think in new PRs we don't have to mimic some concepts, but we should aim for best quality. If there is no case in which that default value is not overriden then remove it please.

Comment (freezy-sk) [reply src/Filter/SkipExistingProducts.php:20]: also that property should be named `$idField` instead of `$skuField`

Comment (vseager) [reply src/Filter/SkipExistingProducts.php]: @bartoszkubicki please review again.

**Review by bartoszkubicki — APPROVED** [review]

**Comment by bartoszkubicki** [comment]
This fix is a part of latest release https://github.com/WeareJH/m2-module-import/releases/tag/2.4.3

**Pull request merged**